htdocs/core/js/lib_head.js.php

```diff
@@ -1321,7 +1321,7 @@ function price2numjs(amount) {
 
 
 jQuery(document).ready(function() {
-	jQuery(".butAction.dropdown-toggle").on("click", function(event) {
+	jQuery(document).on("click", ".butAction.dropdown-toggle", function(event) {
```

Comment:
This this ajax load button.

Reply:
> This looks better than compensating with js code to fix a bug in php.

@eldy this javascript is an other fix (the 4th) ... sorry I haven't explain enouth this one, it's for buttons created by js or ajax loaded

I updated the fix description